doc/REST-interface.md

```diff
@@ -72,7 +72,6 @@ Only supports JSON as output format.
 * pruned : (boolean) if the blocks are subject to pruning
 * pruneheight : (numeric) highest block available
 * softforks : (array) status of softforks in progress
-* bip9_softforks : (object) status of BIP9 softforks in progress
```

Comment:
19390: too early for this, we still have `bip9_softforks`

Reply:
dropped 19390